act-rules-format.bs

```diff
@@ -15,83 +15,83 @@ Markup Shorthands: markdown yes
 
-Not all atomic rules have to be part of a composite rule. Composite rules are used when the outcomes of multiple rules need to be combined in order to determine (non-)conformance of a test subject to an [=accessibility requirement=].
+The separation between atomic rules and composite rules creates a division of responsibilities. Atomic rules test if web content is correctly implemented in a particular solution. Composite rules can test if a combination of [=findings=] from other rules satisfy the accessibility requirement, in part or as a whole.
 
 
 ACT Rule Structure {#act-rule-structure}
 ===============================
 
 An ACT Rule MUST consist of at least the following items:
 
-* Descriptive Title
+* <dfn>Descriptive Title</dfn>
 * [Rule Identifier](#rule-identifier)
 * [Rule Description](#rule-description)
 * [Rule type](#rule-types)
 * [Accessibility Requirements Mapping](#accessibility-requirements-mapping)
-* [Aspects Under Test](#input-aspects) (for atomic rules) OR [Input Rules List](#input-rules-list) (for composite rules)
+* [Rule Input](#input), which is one of the following:
+    * [Input Aspects](#input-aspects) (for atomic rules) OR 
+    * [Input Rules](#input-rules) (for composite rules)
 * [Applicability](#applicability)
 * [Expectations](#expectations)
 * [Assumptions](#assumptions)
 * [Accessibility Support](#accessibility-support)
 * [Test Cases](#test-cases)
 * [Change Log](#change-log)
-* [Issues List](#issues-list)
-* [Acknowledgements](#acknowledgements) (optional)
+* [Glossary](#glossary)
+* [Issues List](#issues-list) (optional)
 * [Background](#background) (optional)
-* ACT Rules Format [=outcome=] definition
+* [Acknowledgements](#acknowledgements) (optional)
 
-ACT Rules MUST be written in a format that conforms to the Web Content Accessibility Guidelines [[WCAG]] or a comparable accessibility standard. ACT Rule [test cases](#test-cases) are allowed to contain inaccessible content. If any test case contains accessibility issues listed in [WCAG 2.1 Section 5.2.5 Non-Interference](https://www.w3.org/TR/WCAG21/#cc5), users MUST be warned of this in advance. Using an accessible format supports people with disabilities. It also makes internationalization of ACT Rules easier.
+The ACT Rules format does not prescribe what format ACT Rules can be written in (e.g. HTML, DOCX, PDF, etc.). However, ACT Rules MUST be written in a document that conforms to the Web Content Accessibility Guidelines [[WCAG]] or a comparable accessibility standard. This ensures that ACT Rules are accessible to people with disabilities. ACT Rule [test cases](#test-cases) are allowed to contain inaccessible content. If any test case contains accessibility issues listed in [WCAG 2.1 Section 5.2.5 Non-Interference](https://www.w3.org/TR/WCAG21/#cc5), users MUST be warned of this in advance. In addition to supporting people with disabilities, using an accessible format also makes internationalization of ACT Rules easier.
```

Comment:
Where should accessibility issues in test cases be documented to meet this "warned in advance" requirement?  In the test case itself, in the rule description? Is that flexible?

Reply:
Also, there is the use of normative language - "MUST" and this is a note, right?  Is this requirement in the ACT Rules Format somewhere?

Reply:
To your latter comment @maryjom, AFAIK this is not a note but a paragraph in a normative section of the document. It actually sets two normative requirements (accessible content and advance warning).